File-Formats/PDF/Working-with-OCR/Features.md

```diff
@@ -1326,4 +1326,12 @@ processor.Settings.Performance = Performance.Fast
 
 {% endhighlight %}
 
-{% endtabs %}  
\ No newline at end of file
+{% endtabs %}  
+## TesseractBinaries paths and Tesseract language data
+
+Starting with v21.2.x, TesseractBinaries and Tesseract language data do not have any reference paths added. However, if you need to refer TesseractBinaries and Tessdata paths manually in your application.
```

Comment:
Starting with v21.1.x, TesseractBinaries and Tesseract language data folder paths are added by default. So, there is no need to provide these paths explicitly. However, you can refer TesseractBinaries and Tessdata paths manually in your application as per the requirement.

Reply:
Updated the content.